fabfile/add.py

```diff
@@ -156,3 +156,10 @@ def tpch(**kwargs):
 def jdbc():
     'Adds everything required to test out the jdbc connecter'
     sudo('yum install -q -y java-1.6.0-openjdk-devel') # we only need java on the master
+
+@task
+@roles('master')
+def add_coordinator():
+    utils.psql("SELECT master_add_node('localhost', {}, groupid => 0);".format(config.PORT))
```

Comment:
the other workers would not be able to reach the coordinator via 'localhost'

Reply:
right, changed